SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -204,66 +204,63 @@ USpatialGameInstance* USpatialNetDriver::GetGameInstance() const
 }
 
 
-void USpatialNetDriver::StartSetupConnectionConfigFromCommandLine(bool& bOutSuccessfullyLoaded, bool& bOutUseReceptionist)
+bool USpatialNetDriver::TrySetupConnectionConfigFromCommandLine()
 {
-	USpatialGameInstance* GameInstance = GetGameInstance();
-
-	FString CommandLineLocatorHost;
-	FParse::Value(FCommandLine::Get(), TEXT("locatorHost"), CommandLineLocatorHost);
-	if (!CommandLineLocatorHost.IsEmpty())
+	FString workerType = GetGameInstance()->GetSpatialWorkerType().ToString();
+	bool bOutSuccessfullyLoaded = Connection->LocatorConfig.TryLoadCommandLineArgs();
+	if (bOutSuccessfullyLoaded)
 	{
-		bOutSuccessfullyLoaded = Connection->LocatorConfig.TryLoadCommandLineArgs();
-		bOutUseReceptionist = false;
+		Connection->SetConnectionType(ESpatialConnectionType::Locator);
+		Connection->LocatorConfig.WorkerType = workerType;
 	}
 	else
 	{
-		bOutSuccessfullyLoaded = Connection->ReceptionistConfig.TryLoadCommandLineArgs();
-		bOutUseReceptionist = true;
+		bOutSuccessfullyLoaded = Connection->DevAuthConfig.TryLoadCommandLineArgs();
```

Comment:
Nit: I am not sure bOutSuccessfullyLoaded is the right name for this variable now. "Out" is associated with reference arguments that get modified.